hugo-site/hugo.toml

```diff
@@ -73,6 +73,12 @@ theme = "freenet"
     name = "Contribute"
     weight = 10
 
+  [[menu.main]]
```

Comment:
I think you will also need to add menu entries for each of the sub-pages, nested under the Manual section